www/views/lockedView.html

```diff
@@ -0,0 +1,21 @@
+<ion-view id="locked-view">
+  <ion-nav-bar class="bar-royal">
+    <ion-nav-title>{{title}}</ion-nav-title>
```

Comment:
I think you should remove the nav bar. This title is not necessary. And also is wrong. Should be a title for the view, not a state.